script/build-newsletter-editions.js

```diff
@@ -93,15 +93,18 @@ async function saveNewsletterEditions(organizationId, letterheadData) {
   for await (let newsletter of letterheadData.items) {
     let headline = shared.cleanContent(newsletter.title);
 
-    if (!newsletter.publicationDate) {
+    if (newsletter.publicationStatus != 1) {
```

Comment:
Since this is Letterhead-specific knowledge, I would add a comment here mentioning the status codes, like you did in the PR description.